reporthandling/apis/statuses.go

```diff
@@ -3,18 +3,17 @@ package apis
 type ScanningStatus string
 
 const (
-	StatusPassed     ScanningStatus = "passed"
-	StatusExcluded   ScanningStatus = "excluded"
-	StatusIgnored    ScanningStatus = "ignored"
-	StatusFailed     ScanningStatus = "failed"
-	StatusSkipped    ScanningStatus = "skipped"
-	StatusUnknown    ScanningStatus = "" // keep this empty
-	StatusIrrelevant ScanningStatus = "irrelevant"
-	StatusError      ScanningStatus = "error"
-)
+	StatusPassed            ScanningStatus = "passed"
```

Comment:
[Q] Is there a reason for the sub-statuses are also of type `ScanningStatus`? 
Since we defined that there are only 3 statuses, this might be confusing.